src/reactapp/src/components/items/components/CartItemsFormManager/CartItemsFormManager.jsx

```diff
@@ -51,14 +51,15 @@ function CartItemsFormManager({ children, formikData }) {
     if (needToPopulateForm && cartItemsAvailable) {
```

Comment:
Applied this change. The suggested approach is what I take normally in my coding inside a function. But in `useEffect`, I take the current approach because the return inside useEffect is special implications. So due to this,  I think it would be good if we can avoid a return in useEffect if possible.

Reply:
The `useEffect` can return a function to cleanup when the component unmounts, like what you'd code into `componentWillUnmount` function when using class based components. In the case where you dont return a function nothing happens and it is perfectly fine syntax use.

Reply:
👍